wordpress-mobile/WordPress-iOS
# ULS: Update Auth to use new 2FA view

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/336
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/337

This updates auth to show the new 2FA view in the unified Google flow. Since this view is not functional, the `unifiedGoogle` feature flag has been disabled.

To test:

---
- Enable the `unifiedGoogle` feature flag.
- Select `Continue with Google` from either login or signup.
- Verify the updated `Waiting for Google` view is displayed:

<kbd>![waiting_for_google](https://user-images.githubusercontent.com/1816888/88594376-4f860180-d01e-11ea-9ced-e96d9dd75e2d.png)</kbd>

---
- Select a Google account that:
  - _Does not_ have a WP password set.
  - _Does_ have WP 2FA enabled.
- Verify the new 2FA view is displayed:

<kbd>![2fa_view](https://user-images.githubusercontent.com/1816888/88595108-a0e2c080-d01f-11ea-8380-262f727c3a99.png)</kbd>

---
PR submission checklist:
